packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -51,6 +51,7 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
     elasticStackGetStarted: isServerless
       ? `${SERVERLESS_DOCS}`
       : `${ELASTIC_WEBSITE_URL}guide/en/index.html`,
+    apiReference: `${ELASTIC_WEBSITE_URL}guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/api-reference.html`,
```

Comment:
>    apiReference: `${ELASTIC_WEBSITE_URL}guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/api-reference.html`,

If you do want this link to take you to a list of all APIs for all products and deployment types, this is a good target for now. If you're only interested in a specific subset of APIs and you want to differentiate between serverless and non-serverless contexts, you could use the "isServerless" condition here. Just lmk if there a specific subset you want to be able to link to and I can help figure out the URL.

Reply:
cc: @joemcelroy @serenachou

Reply:
IMO, we could go with stack url and revisit in future if need to differentiate between serverless & stack. 
@joemcelroy please lmk if you think otherwise